Zend/zend_smart_str.h

```diff
@@ -25,6 +25,10 @@
 	smart_str_appendl_ex((dest), (src), strlen(src), (what))
 #define smart_str_appends(dest, src) \
 	smart_str_appendl((dest), (src), strlen(src))
+#define smart_str_trim_to_len(dest) \
+	smart_str_trim_to_len_ex((dest), 0)
+#define smart_str_finalize(dest) \
```

Comment:
while `smart_str_trim_to_len` name is right on point, I think however `smart_str_finalize` might be slightly misleading for what it truly does, just personal opinion. But definitively a nice-to-have for 8.2

Reply:
sure we could go ahead with.